test_cases/arc4_dynamic_arrays/out/DynamicArrayContract.approval.teal

```diff
@@ -463,7 +463,7 @@ get_string3:
     proto 0 1
     // arc4_dynamic_arrays/contract.py:185
     // return arc4.String("Hello World")
-    pushbytes 0x000b48656c6c6f20576f726c64
+    pushbytes "\x00\x0bHello World"
```

Comment:
Since this value is a mix of UTF8 and escaped hex, might be better to keep the original base16 encoding.

The source comment provides the context of the UT8 value already